src/xenia/kernel/xboxkrnl/xboxkrnl_strings.cc

```diff
@@ -290,7 +293,7 @@ int32_t format_core(PPCContext* ppc_context, FormatData& data, ArgList& args,
           state = FS_Type;
           continue;
         } else if (c == 'w') {
-          flags |= FF_IsWide;
```

Comment:
With the rename I mentioned earlier, we want to set `FF_IsWide` here, so you can remove this changed line.